db/models/storeMongo.js

```diff
@@ -31,7 +31,7 @@ const storeSchema = mongoose.Schema({
 
 const Store = mongoose.model('Store', storeSchema);
 
-const findOne = id => Store.findOne({ place_id: id });
+const findOne = id => Store.findOne({ place_id: id }).lean();
```

Comment:
I noticed greater performance with find() instead of findOne().  Could be something to explore.

Reply:
I'll test this out, and let you know if it makes a difference.

Reply:
I just tested, it seems to make no noticable difference (61K RPM with findOne() vs 60K RPM with find()). Keeping findOne for now.